vpr/src/util/vpr_utils.h

```diff
@@ -194,12 +194,33 @@ t_logical_block_type_ptr pick_best_logical_type(t_physical_tile_type_ptr physica
 //the best expected physical tile the block should use (if no valid placement).
 t_physical_tile_type_ptr get_physical_tile_type(const ClusterBlockId blk);
 
+//Returns the sub tile index (within 'physical_tile') corresponding to the
+//'logical block'
+int get_logical_block_physical_sub_tile_index(t_physical_tile_type_ptr physical_tile,
+                                              t_logical_block_type_ptr logical_block);
+
+//Returns the physical pin index (within 'physical_tile') corresponding to the
+//logical index ('pin' of the first instance of 'logical_block' within the physcial tile.
+//
+//Throws an exception if the corresponding physical pin can't be found.
+int get_physical_pin(t_physical_tile_type_ptr physical_tile,
```

Comment:
This function is incorrect, as it can only return a pin from the first instance of a logical tile in a physical tile.  I'm not sure under what circumstances this is ever the right thing to do?